aspnetcore/web-api/http-repl.md

```diff
@@ -809,7 +809,22 @@ To set an HTTP request header, use one of the following approaches:
 
 ## Test secured endpoints
 
-The HTTP REPL supports the testing of secured endpoints through the use of HTTP request headers. Examples of supported authentication and authorization schemes include basic authentication, JWT bearer tokens, and digest authentication. For example, you can send a bearer token to an endpoint with the following command:
+The HTTP REPL supports the testing of secured endpoints in two ways: via the default credentials of the logged in user or through the use of HTTP request headers. 
```

Comment:
Looks good.  Would consider adding wording this is helpful for Windows Auth.  Maybe: 

"...via the default credentials of the logged in user _(for instance if you are are using Windows Auth with IIS)_